docs/main/ios/spm.md

```diff
@@ -14,21 +15,20 @@ Since Capacitor 6, you can choose between using CocoaPods or Swift Package Manag
 
 We've tried our best to make sure you don't have to change much about how you work with Capacitor to use SPM, but there are a few things to understand.
 
-### How it works
+## How it works
 
 When a Capacitor project is using SPM we use a 'Base SPM' package that will serve as the place that references all of your projects dependencies:
 
 ![Base SPM Picture](../../../static/img/v6/docs/ios/spm/base-spm.png)
 
 The Capacitor CLI will modify the CapAPP-SPM package when you sync new plugins. It is important you do not touch the contents here because the CLI can and will change things.
```

Comment:
```suggestion
The Capacitor CLI will modify the CapApp-SPM package when you sync new plugins. It is important you do not touch the contents here because the CLI can and will change things.
```